[PS3] - Multipass shaders doesn't work

I got `[ERROR] [GL]: Failed to set up frame buffer objects. Multi-pass shading will not work.`
Catched error on first loop with` i = 0`:

```
static bool gl2_create_fbo_targets(gl_t *gl, gl2_renderchain_data_t *chain)
{
   unsigned i;

   glBindTexture(GL_TEXTURE_2D, 0);
   gl2_gen_fb(chain->fbo_pass, chain->fbo);

   for (i = 0; i < (unsigned)chain->fbo_pass; i++)
   {
      GLenum status;
      gl2_bind_fb(chain->fbo[i]);
      gl2_fb_texture_2d(RARCH_GL_FRAMEBUFFER,
            RARCH_GL_COLOR_ATTACHMENT0, GL_TEXTURE_2D, chain->fbo_texture[i], 0);

      status = gl2_check_fb_status(RARCH_GL_FRAMEBUFFER);
      if (status != RARCH_GL_FRAMEBUFFER_COMPLETE) 
         goto error;
	  
   }

```

In the past, when there was `ps3\gcmgl` folder it worked (multipass).....we are talking about Retroarch 1.0.0.2 - 1..0.0.3
At least help me to understand where search.....

--- a/gfx/drivers/gl.c
+++ b/gfx/drivers/gl.c
@@ -857,11 +857,16 @@ static void gl2_create_fbo_texture(gl_t *gl,
       else
 #endif
       {
-#if defined(HAVE_OPENGLES2) || defined(HAVE_PSGL)
+#if defined(HAVE_OPENGLES2)
          glTexImage2D(GL_TEXTURE_2D,
                0, GL_RGBA,
                gl->fbo_rect[i].width, gl->fbo_rect[i].height, 0,
                GL_RGBA, GL_UNSIGNED_BYTE, NULL);
+#elif defined(HAVE_PSGL)
+         glTexImage2D(GL_TEXTURE_2D,
+               0, GL_ARGB_SCE,
+               gl->fbo_rect[i].width, gl->fbo_rect[i].height, 0,
+               GL_ARGB_SCE, GL_UNSIGNED_BYTE, NULL);
 #else
          /* Avoid potential performance
           * reductions on particular platforms. */

